isovar/assembly.py

```diff
@@ -185,9 +178,9 @@ def iterative_assembly(
 
```

Comment:
@ihodes I'm worried about "false" assemblies that only overlap on the variant nucleotides (but actually come from different splice isoforms). It seems that requiring some amount of overlap beyond the variant nucleotides will decrease the chances of false assembly but the exact amount (30) is pretty arbitrarily chosen: shorter than a read length but long enough to be a kmer unlikely to happen by chance.